test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -485,11 +491,16 @@ describe("GET tests", () => {
   });
 
   test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with trust data and feature flag on`, async () => {
+    mockIsActiveFeature.mockReturnValueOnce(true); // FEATURE FLAG REDIS REMOVAL
```

Comment:
do we know for sure if these mockIsActive feature are for redis removal flags? if not sure I would remove the comments as they would be misleading.

Reply:
Best would be to find out what these calls are for and comment appropriately. I think I've done this before by coding a 'temp' mock implementation that outputs the feature flag name before returning the value.

Reply:
`mockIsActiveFeature` is called every time the `isActiveFeature` method is invoked, and in this case, it's for the `FEATURE_FLAG_ENABLE_REDIS_REMOVAL` and `FEATURE_FLAG_ENABLE_TRUSTS_WEB` only.

Have added a comment at the top specifying that it's for these two flags.